src/eslint-config-adeira/src/presets/react.js

```diff
@@ -211,6 +212,40 @@ module.exports = ({
     'jsx-a11y/scope': ERROR,
     'jsx-a11y/tabindex-no-positive': ERROR,
   },
+  overrides: [
+    {
+      // React Testing Library (https://github.com/testing-library/eslint-plugin-testing-library)
+      files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
+      rules: {
+        'testing-library/await-async-query': OFF, // TODO: seems to be broken for some `findAllByProps` in our codebase
```

Comment:
Interesting, did you create an issue on their github?

Reply:
I did not. I didn't have much time for it but most importantly it was not very obvious to me what's the problem and if it's even a problem with `testing-library`. I thought it's going to be better to move forward and disable this for now and return to it later (since I am going to be enabling more rules and improving the plugin setup in general).